packages/graph/hash_graph/lib/graph/src/datastore/postgres/mod.rs

```diff
@@ -1,20 +1,27 @@
+mod row_types;
+
 use async_trait::async_trait;
 use error_stack::{IntoReport, Result, ResultExt};
+use row_types::DataTypeRow;
 use sqlx::PgPool;
+use uuid::Uuid;
 
-use crate::datastore::{DatabaseConnectionInfo, Datastore, DatastoreError};
+use crate::{
+    datastore::{DatabaseConnectionInfo, Datastore, DatastoreError},
+    types::{AccountId, BaseId, DataType, Identifier, Qualified, VersionId},
+};
 
 /// A Postgres-backed Datastore
 pub struct PostgresDatabase {
     pub pool: PgPool,
 }
 
 impl PostgresDatabase {
-    /// Creates a new `PostgresDatabase` object
+    /// Creates a new `PostgresDatabase` object.
     ///
     /// # Errors
     ///
-    /// If creating a [`PgPool`] connection returns an error
+    /// If creating a [`PgPool`] connection returns an error.
```

Comment:
```suggestion
    /// - [sqcx `Error`], if creating a [`PgPool`] connection returns an error.
    ///
    /// [sqcx `Error`]: sqlx::Error
```

Reply:
sqlx* Not sqcx.

I also think we should probably return `DatastoreError` here, or something like `PostgresDbError`

Reply:
Changed this to propagate `DatastoreError` instead